src/Gaffer/ApplicationRoot.cpp

```diff
@@ -130,7 +130,8 @@ std::string ApplicationRoot::preferencesLocation() const
 	}
 
 	std::string result = home;
-	result += "/gaffer/startup/" + getName().string();
+
+	result = FileSystemPath( result + "/gaffer/startup/" + getName().string() ).string();
```

Comment:
I don't think we should need to construct a FileSystemPath to do these sorts of conversions - it's intended as a persistent object for navigating a filesystem using the Path API, not a utility class.

Reply:
Fixed in a number of places, and I think should be cleaner now.